tools/auto_tuner/gen/generate_combinations.py

```diff
@@ -29,6 +29,7 @@
     "interleaved": "::blas::gemm_batch_type_t::interleaved",
 }
 
+
```

Comment:
We should probably format all Python files at some point. Which formatter did you use? We generally use yapf in other projects, though that's not universal at all.

Reply:
Ah I think ```autopep8``` was used as that's the vscode default, I can reformat with yapf though if that's better.

Reply:
Actually I was mistaken! The formatter in this case is ```yapf```